### ISSUE TITLE
Black Output --- Image2Image plugin not working and Render On Changes not updating

### ISSUE THREAD

**Issue by farhang0** [NimaNzrii/comfyui-photoshop#72]
### Hi @NimaNzrii,

https://github.com/NimaNzrii/comfyui-photoshop/assets/163278907/4e39400b-1799-4b97-b452-01d7902327c0

I hope this message finds you well. 👍 

I wanted to reach out to you to express my gratitude for your work on the ComfyUI plugin. I've been trying to use it to generate images from text and text from images, but I'm running into some issues.

### As you can see in the video I've attached, the Text2Image portion of the plugin seems to be working fine. **However**, when I try to use **Image2Image, it only generates a black image**. Additionally, when I enable the **"Render On Changes"** option, nothing updates when I make changes to the components on the page. It seems like there's no communication happening between the plugin and the UI.

I've tried **reinstalling and setting up** ComfyUI from scratch twice now, but I'm still having the same problems. I'm not receiving any errors, aside from a warning that appears when ComfyUI starts up (which is also shown in the video and attached to this message).

I'm hoping you might be able to shed some light on this issue and help me get ComfyUI working properly. I'm really excited about the potential of this plugin, and I would be very grateful for your assistance.

Thank you for your time and consideration.

Sincerely,
Far

```
E:\SOFTWARE\ComfyUI_windows_portable>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build
## ComfyUI-Manager: installing dependencies done.
** ComfyUI startup time: 2024-05-26 14:32:11.913617
** Platform: Windows
** Python version: 3.11.6 (tags/v3.11.6:8b6ee5b, Oct  2 2023, 14:57:12) [MSC v.1935 64 bit (AMD64)]
** Python executable: E:\SOFTWARE\ComfyUI_windows_portable\python_embeded\python.exe
** Log path: E:\SOFTWARE\ComfyUI_windows_portable\comfyui.log

Prestartup times for custom nodes:
   0.0 seconds: E:\SOFTWARE\ComfyUI_windows_portable\ComfyUI\custom_nodes\rgthree-comfy
   0.4 seconds: E:\SOFTWARE\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

Total VRAM 8192 MB, total RAM 16282 MB
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3060 Ti : cudaMallocAsync
VAE dtype: torch.bfloat16
Using pytorch cross attention
[ComfyUI-Inference-Core-Nodes.src.inference_core_nodes.controlnet_preprocessors] | INFO -> Using ckpts path: E:\SOFTWARE\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Inference-Core-Nodes\src\inference_core_nodes\controlnet_preprocessors\ckpts
[ComfyUI-Inference-Core-Nodes.src.inference_core_nodes.controlnet_preprocessors] | INFO -> Using symlinks: False
[ComfyUI-Inference-Core-Nodes.src.inference_core_nodes.controlnet_preprocessors] | INFO -> Using ort providers: ['CUDAExecutionProvider', 'DirectMLExecutionProvider', 'OpenVINOExecutionProvider', 'ROCMExecutionProvider', 'CPUExecutionProvider', 'CoreMLExecutionProvider']
DWPose: Onnxruntime with acceleration providers detected
### Loading: ComfyUI-Manager (V2.34)
### ComfyUI Revision: 2185 [58f83880] | Released on '2024-05-16'
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json_PS_ All packages are installed.
_PS_ backend_path E:\SOFTWARE\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-photoshop\Backend.py
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
_PS_ Backend Started
[comfyui_controlnet_aux] | INFO -> Using ckpts path: E:\SOFTWARE\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts
[comfyui_controlnet_aux] | INFO -> Using symlinks: False
[comfyui_controlnet_aux] | INFO -> Using ort providers: ['CUDAExecutionProvider', 'DirectMLExecutionProvider', 'OpenVINOExecutionProvider', 'ROCMExecutionProvider', 'CPUExecutionProvider', 'CoreMLExecutionProvider']
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
_PS_ Photoshop launched('127.0.0.1', 55980)
_PS_ comfyUi Not Connected
[comfy_mtb] | INFO -> loaded 73 nodes successfuly
[comfy_mtb] | INFO -> Some nodes (5) could not be loaded. This can be ignored, but go to http://127.0.0.1:8188/mtb if you want more information.

[rgthree] Loaded 39 extraordinary nodes.
```
Adobe Sysinfo :

```
Adobe Photoshop Version: 24.2.0 20230218.r.315 28c9f72  x64
Number of Launches: 52
Operating System: Windows 10 64-bit
Version: 10 10.0.19045.4355
System architecture: Intel CPU Family:6, Model:5, Stepping:3 with MMX, SSE Integer, SSE FP, SSE2, SSE3, SSE4.1, SSE4.2, AVX, AVX2, HyperThreading
```

**Comment by FreezaSama** [comment]
Goes without saying Nima is the boss. but I too have this problem. usually it starts ok but after a few changes photoshop stops communicating with comfy. I'll be honest... I am VERY surprised Adobe is even letting this go through as it's EXACTLY what photoshop should have implemented.

**Comment by locsmax** [comment]
https://github.com/NimaNzrii/comfyui-photoshop/issues/85#issuecomment-2152149278

**Comment by loztm** [comment]
> Goes without saying Nima is the boss. but I too have this problem. usually it starts ok but after a few changes photoshop stops communicating with comfy. I'll be honest... I am VERY surprised Adobe is even letting this go through as it's EXACTLY what photoshop should have implemented.

Same problem here... any solution?

**Comment by NimaNzrii** [comment]
# Fixed in V1.5.0 ✨
this problem should be fixed in ver 1.5.0
please update plugin and the node and let me know if it helped
